<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KinectViewer</class>
 <widget class="QMainWindow" name="KinectViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1105</width>
    <height>499</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>5000</width>
    <height>5000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>KinectViewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QVTKWidget" name="qvtkWidget" native="true">
    <property name="geometry">
     <rect>
      <x>136</x>
      <y>10</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStopStream">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>436</y>
      <width>121</width>
      <height>57</height>
     </rect>
    </property>
    <property name="text">
     <string>Stream running</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnResetCamera">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>376</y>
      <width>121</width>
      <height>57</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoom to fit</string>
    </property>
   </widget>
   <widget class="QSlider" name="sldClipDistance">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>71</y>
      <width>65</width>
      <height>277</height>
     </rect>
    </property>
    <property name="maximum">
     <number>3000</number>
    </property>
    <property name="singleStep">
     <number>50</number>
    </property>
    <property name="sliderPosition">
     <number>3000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLabel" name="lblClipDistance">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>8</y>
      <width>85</width>
      <height>53</height>
     </rect>
    </property>
    <property name="text">
     <string>Clip far data [mm]</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblRGBimage">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>10</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTrackFace">
    <property name="geometry">
     <rect>
      <x>892</x>
      <y>268</y>
      <width>109</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>Track face</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
